### PR TITLE
How usage affects job priority within an instance

### LINKED ISSUE #1863: Importance of paying attention to usage on instances
From @francabrera:

Since they may have limits and because how fair-share works and link them to the fair-share docs from there, the request is to give more info on instance limits and how usage affects priority.

Points from Fran:

- You share access to the instance with others, so others may be causing usage in the instance besides of you
- Your usage and others usage are accounted on your instance allocation
- if the instance is limited and exceeded its allocation, your jobs will not be processed until more time is available.
- if the instance is not limited if their allocation is exceeded your jobs may experience longer queue times
- how fair-share works in terms of usage/allocation ratio (this could be brief and link to the fair-share scheduler page)
- users should understand their usage may be affecting others and others usage may be affecting them, by affecting it means longer queue times or even reaching their limits and not being able to run anything until more time is available (28-day rolling window fair-share)

Here's what users see on the dashboard:

image

### PR DESCRIPTION
**Pull request by abbycross**
Closes #1863

### CODE REVIEW

**Review by francabrera — APPROVED** [review]

File: docs/guides/instances.mdx [hunk 1]
```diff
@@ -47,6 +47,22 @@ By default, users who sign up for an IBM Quantum account are assigned to the Ope
 
 Those using the Open Plan instance have up to 10 minutes total of quantum time per month, which resets at 00:00:00 UTC on the first of each calendar month. Open Plan users can track their usage on the [Platform dashboard](https://quantum.ibm.com/), [Jobs](https://quantum.ibm.com/jobs), and [Account](https://quantum.ibm.com/account) pages.
 
+## How usage affects job priority within an instance
+
+*This section applies to Premium Plan users only, since their access includes multiple instances.* When determining which jobs to run, the [fair-share scheduler](./fair-share-scheduler) takes into account an instance's usage of the QPU in question. For example, an instance that has already run many jobs on a certain QPU might not have as high a priority on that QPU, when compared to an instance that has very low usage.
+
+The jobs you run, as well as the jobs run by other collaborators in the same instance, count toward the reported usage for that instance. You can see the usage for a selected instance on your [dashboard](https://quantum.ibm.com/) (use the [instance switcher](#switch-instances) at the top of the page to change which instance is reflected on your dashboard).
```
Comment (beckykd) [inline docs/guides/instances.mdx:54]:
```suggestion
The jobs you run and the jobs run by other collaborators in the same instance count toward the reported usage for that instance. You can see the usage for an instance on your [dashboard](https://quantum.ibm.com/) (use the [instance switcher](#switch-instances) at the top of the page to change which instance is reflected on your dashboard).  
```

File: docs/guides/instances.mdx [hunk 2]
```diff
@@ -47,6 +47,22 @@ By default, users who sign up for an IBM Quantum account are assigned to the Ope
...
+
+The jobs you run, as well as the jobs run by other collaborators in the same instance, count toward the reported usage for that instance. You can see the usage for a selected instance on your [dashboard](https://quantum.ibm.com/) (use the [instance switcher](#switch-instances) at the top of the page to change which instance is reflected on your dashboard).  
+
+Be aware that if an instance has been marked "limited" by your administrator and the instance exceeds its allocation (also defined by your administrator) within the 28-day rolling window, jobs run with that instance will not complete until more time is available. If an instance is not limited, and if the instance exceeds its allocation, jobs run with that instance are likely to run at a lower priority and experience longer queue times.
```
Comment (beckykd) [inline docs/guides/instances.mdx:56]:
```suggestion
If an instance has been marked as "limited" by your administrator (you will see a "**Remaining**" column for the usage) and the instance exceeds its allocation (defined by your administrator) within the 28-day rolling window, jobs run with that instance will not complete until more time is available. If an instance is not limited and the instance exceeds its allocation, jobs run with that instance are likely to run at a lower priority and experience longer queue times.
```

File: docs/guides/instances.mdx [hunk 3]
```diff
@@ -47,6 +47,22 @@ By default, users who sign up for an IBM Quantum account are assigned to the Ope
...
+
+An alert will display on an instance's usage when it has exceeded its allocation.
```
Comment (beckykd) [inline docs/guides/instances.mdx:58]:
```suggestion
An alert will display on an instance's usage when it has exceeded its allocation.
```
```suggestion
An alert displays on an instance's usage when it has exceeded its allocation.
```

**Review by beckykd — COMMENTED** [review]
Just some minor comments

**Comment by beckykd** [comment]
I'm not sure about where this section lives. (https://qiskit.github.io/documentation/pr-1872/guides/instances#instances)  Would it be better indented under Instances and jobs?  Thoughts @Eric-Arellano ?
image

**Review by beckykd — APPROVED** [review]

**Pull request merged**